src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private int radius;
+
+    public Circle(String color, int radius) {
```

Comment:
When creating a Circle, consider using separate variables for the radius value instead of passing the expression directly to the constructor for better readability.